WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -356,10 +356,10 @@ private extension TotalsView {
         static let totalAmountFont: POSFontStyle = .posHeading
         static let separatorColor: Color = Color.posOutlineVariant
 
-        static let shimmeringCornerRadius: CGFloat = 4
-        static let shimmeringWidth: CGFloat = 334
+        static let shimmeringCornerRadius: CGFloat = 8
+        static let shimmeringWidth: CGFloat = 342
```

Comment:
I'm thinking if setting a constant width was the best choice in the past but let's leave it like that for now.

Reply:
Yes, I was thinking about setting a percentage-based width for this. But I thought I would keep the scope minimal before the release.